charts/rancher-istio/1.5.901/files/injection-template.yaml

```diff
@@ -348,15 +348,24 @@ containers:
     runAsNonRoot: true
     runAsUser: 1337
     {{- end }}
-  resources:
-    {{ if or (isset .ObjectMeta.Annotations `sidecar.istio.io/proxyCPU`) (isset .ObjectMeta.Annotations `sidecar.istio.io/proxyMemory`) -}}
-    requests:
-      {{ if (isset .ObjectMeta.Annotations `sidecar.istio.io/proxyCPU`) -}}
-      cpu: "{{ index .ObjectMeta.Annotations `sidecar.istio.io/proxyCPU` }}"
-      {{ end}}
-      {{ if (isset .ObjectMeta.Annotations `sidecar.istio.io/proxyMemory`) -}}
-      memory: "{{ index .ObjectMeta.Annotations `sidecar.istio.io/proxyMemory` }}"
-      {{ end }}
+resources:
```

Comment:
formatting looks off - this will cause an issue , unless you confirmed and its just a github ui issue.